platform-tests/src/platform/acceptance/uaa_styling_test.go

```diff
@@ -0,0 +1,71 @@
+package acceptance_test
+
+import (
+	"encoding/json"
+	"github.com/PuerkitoBio/goquery"
+	"github.com/cloudfoundry-incubator/cf-test-helpers/cf"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+	"io/ioutil"
+	"net/url"
+)
+
+var _ = Describe("UAA Styling", func() {
+	var uaaURL *url.URL
+	var uaaLoginURL url.URL
+	var uaaStylesheetURL url.URL
+	var customStylesheetURL url.URL
+
+	BeforeEach(func() {
+		infoCommand := cf.Cf("curl", "/v2/info")
+		Expect(infoCommand.Wait(testConfig.DefaultTimeoutDuration())).To(Exit(0))
+
+		var infoResp struct {
+			TokenEndpoint string `json:"token_endpoint"`
+		}
+		err := json.Unmarshal(infoCommand.Buffer().Contents(), &infoResp)
+		Expect(err).NotTo(HaveOccurred())
+
+		uaaURL, err = url.Parse(infoResp.TokenEndpoint)
+		Expect(err).NotTo(HaveOccurred())
+
+		uaaLoginURL = *uaaURL
+		uaaLoginURL.Path = "/login"
+
+		uaaStylesheetURL = *uaaURL
+		uaaStylesheetURL.Path = "/resources/oss/stylesheets/application.css"
+
+		customStylesheetURL = url.URL{
+			Scheme: "https",
+			Host:   "paas-uaa-assets." + testConfig.AppsDomain,
+			Path:   "/stylesheets/application.css",
+		}
+	})
+
+	It("should be using the custom stylesheet", func() {
+		response, err := httpClient.Get(uaaLoginURL.String())
+		Expect(err).NotTo(HaveOccurred())
+		uaaLoginDoc, err := goquery.NewDocumentFromResponse(response)
+		Expect(err).NotTo(HaveOccurred())
+
+		stylesheetHrefs := uaaLoginDoc.Find("link[rel=stylesheet]").Map(func(_ int, linkTag *goquery.Selection) string {
+			href, _ := linkTag.Attr("href")
+			return href
+		})
+		Expect(stylesheetHrefs).To(HaveLen(2))
```

Comment:
Very minor, but you might be interested to know that you can simplify these 3 assertions with the [`ConsistOf` matcher](https://godoc.org/github.com/onsi/gomega#ConsistOf) if the order doesn't matter:
```
Expect(stylesheetHrefs).To(ConsistOf([]string{
    "/vendor/font-awesome/css/font-awesome.min.css",
    customStylesheetURL.String(),
}))
```
If the order does matter then `Equal` will do.

Reply:
Awesome. Thanks Dan!